engines/order_management/spec/services/order_management/subscriptions/proxy_order_syncer_spec.rb

```diff
@@ -210,7 +228,19 @@ module Subscriptions
 
                 context "and the oc is open and closes between begins_at and ends_at" do
                   let(:oc) { open_oc }
+
                   it "keeps the proxy order" do
+                    schedule = create(:schedule, order_cycles: [oc])
+                    subscription = build(
+                      :subscription,
+                      begins_at: now + 1.minute,
+                      ends_at: now + 2.minutes,
+                      schedule: schedule
+                    )
+
+                    syncer = ProxyOrderSyncer.new(subscription)
+                    syncer.sync!
```

Comment:
I am sorry I don't understand this copy pasted code.
For example here, under the context "and the proxy order has not already been placed" there are 7 specs, 3 of them need this exact same setup, why not group these 3 in a new sub-context, give it some nice name, and have this code once in a before block?